src/robusta/core/reporting/base.py

```diff
@@ -169,7 +173,12 @@ def matches(self, match_requirements: Dict[str, Union[str, List[str]]], scope_re
         # 1. "scope" check
         accept = True
         if scope_requirements is not None:
-            matcher = FilterableScopeMatcher(self.attribute_map)
+            data = self.attribute_map
```

Comment:
I think it would be best to load and cache the namespaces, only if there's a matcher that uses it

Reply:
as discussed, always listing the namespaces does not incur an unacceptable time penalty, so we leave it